packages/nimbus/src/components/rich-text-input/rich-text-input.mdx

```diff
@@ -2,185 +2,197 @@
-The rich text input supports standard keyboard shortcuts for formatting:
-
-- **Cmd/Ctrl + B** - Toggle bold formatting
-- **Cmd/Ctrl + I** - Toggle italic formatting
-- **Cmd/Ctrl + U** - Toggle underline formatting
-- **Cmd/Ctrl + `** - Toggle code formatting
-- **Shift + Enter** - Insert soft line break
+## Guidelines
+
+These guidelines provide comprehensive direction for designing and developing
+rich text inputs within Nimbus.
+
+### Best practices
+
+- **Make responsive changes:** Apply changes immediately to the selected text
+  upon clicking a toolbar control.
+- **Utilize tooltips and labels:** Include tooltips for all toolbar actions to
+  enhance discoverability and accessibility.
+- **Keep a user's focus:** Maintain the user's focus within the text area after
+  an action or toggle is selected.
+- **Allow for common shortcuts:** Support standard keyboard shortcuts for
+  formatting (e.g., Ctrl/Cmd+B for bold).
+- **Make it easier to correct mistakes:** Offer an undo/redo history for content
+  changes, and allow for a large enough input window so a user can quickly read
+  content without needing to scroll line by line.
+- **Tailor tooling:** Determine the required formatting options based on the
+  user's needs for the specific application.
+- **Curate tools into meaningful similar collections:** Group related controls
+  logically in the toolbar (e.g., text styling, lists, media).
 
-## Formatting options
-
-### Text formatting
-
-- **Bold** - Make text bold
-- **Italic** - Make text italic
-- **Underline** - Add underline to text
-- **Strikethrough** - Add strikethrough to text
-- **Code** - Format text as inline code
-- **Superscript** - Format text as superscript
-- **Subscript** - Format text as subscript
-
-### Block formatting
-
-- **Headings** - H1, H2, H3, H4, H5 heading levels
-- **Blockquote** - Format text as a blockquote
-- **Bulleted List** - Create unordered lists
-- **Numbered List** - Create ordered lists
-
-### Links
-
-- **Automatic Detection** - URLs are automatically converted to links
-- **Manual Creation** - Use the toolbar to create links
-
-## API Reference
+## Usage
 
-### Props
+Rich text inputs should be used in areas where a user knows that a longer
+response or description is expected.
+
+### When to use
+
+> [!TIP]\
+> When to use
+>
+> - **Editing long-form content:** Such as product descriptions, knowledge base
+>   articles, or email bodies, where visual hierarchy and style are necessary
+>   for readability.
+> - **Formatting text:** Such as applying bold, italics, underlines, or specific
+>   heading levels.
+> - **Structuring content:** With bulleted or numbered lists.
+> - **Embedding external elements:** Like links, images, or videos within other
+>   content.
+
+### When not to use
+
+> [!CAUTION]\
+> When not to use
+>
+> - **Short answers:** Or single-line inputs, such as names, titles, or search
+>   queries.
+> - **Specific data entry:** Unformatted text is required, like SKUs, passwords,
```

Comment:
The Figma says 

<img width="654" height="33" alt="Image" src="https://github.com/user-attachments/assets/9493a841-8c66-4a48-b4a8-06b287dbeced" />

Reply:
@ellelevy Is this a typo or intentional?